extensions/gmsa-dc/v1/Promote-DC.ps1

```diff
@@ -0,0 +1,51 @@
+###################################################################################

```

Comment:
General nit on this script and all the ones below: would be good to add the usual
```
Set-StrictMode -Version Latest
$ErrorActionPreference = 'Stop'
$PSDefaultParameterValues['*:ErrorAction'] = 'Stop'
```
to exit early on errors - easier to catch and debug issues.

Reply:
Thank you for the feedback.  These have been added.